JSDemos/Demos/DataGrid/CustomSummaries/description.md

```diff
@@ -1 +1,13 @@
-The DataGrid support custom summary calculation. In&nbsp;this example, summaries are only calculated against selected grid rows. This behavior is&nbsp;implemented using the [calculateCustomSummary](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/summary/#calculateCustomSummary) and [onSelectionChanged](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/#onSelectionChanged) properties.
\ No newline at end of file
+DataGrid can apply custom server-side and client-side functions to calculate total and group summaries. If the dataset is small, you should choose the client-side implementation.
```

Comment:
> If the dataset is small, you should choose the client-side implementation.

I'm not sure saying that one *should* choose the client-side implementation in this case is correct. Client-side calculation is okay on small datasets, but so is server-side calculation.